brave/browser-laptop
# Adding context menu item to history and bookmark entries for publisher list inclusion

Fixes: #6547

## Submitter Checklist:


## Test Plan:

1. Enable payments
2. Set minimum payment settings a bit high, (Ex: 1 minute, 5 visits)
3. Browse a few different sites to add history entries.
4. Bookmark a few sites naturally via the menu, create some custom bookmarks as well.
5. Create at least one bookmark with a bogus location like `asldjfoasidj`
6. Navigate to `about:bookmarks`
7.  For entries that have not been added to the ledger table, right click on them and confirm that "Add To Publisher List" is enabled and on the context menu. 
8. For the bogus entry, confirm that "Add To Publisher List" is disabled
9.  Use the context menu item to add a couple of the sites to the ledger.
10. Confirm that the items are present in the ledger table, and are set with the minimum visits and visit time set in payment settings.
11. Confirm that those items can not be double-added/stacked with the new context menu item.
12. Repeat steps 7 - 11 with the entries under `about:history`

## Reviewer Checklist:


Tests
